client/src/components/InterpreterSignUpForm.js

```diff
@@ -95,18 +95,14 @@ class InterpreterSignUpForm extends Component {
   render() {
     return (
       <div>
-        <Grid centered column={16}>
-          <Grid.Column width={6}>
+        <Grid padded="horizontally" centered column={16}>
+          <Grid.Column computer="8" mobile="16" tablet="8" width={6}>
```

Comment:
Try this also: `largeScreen="6" computer="10" mobile="16" tablet="10"`

Reply:
it is done